client/extensions/zoninator/state/locks/test/selectors.js

```diff
@@ -0,0 +1,258 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { cloneDeep, tap } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getLock,
+	getLockTimeCreated,
+	getMaxLockPeriod,
+	isBlocked,
+	isExpired,
+	isRequesting,
+} from '../selectors';
+
+describe( 'selectors', () => {
+	const primarySiteId = 123;
+	const primaryZoneId = 456;
+	const secondarySiteId = 789;
+	const secondaryZoneId = 987;
+
+	const emptyState = {
+		extensions: {
+			zoninator: {
+				locks: undefined,
+			},
+		},
+	};
+
+	describe( 'getLock()', () => {
+		const state = tap( cloneDeep( emptyState ), stateTree => {
```

Comment:
Since this is a test, I'd suggest defining `state` explicitly (as opposed to that `tap`/`cloneDeep`/modifying function combination -- looks a bit fragile, TBH).